src/net_setup.c

```diff
@@ -231,10 +232,13 @@ char *get_name(void) {
 }
 
 bool setup_myself_reloadable(void) {
-	free(scriptinterpreter);
-	scriptinterpreter = NULL;
-
-	get_config_string(lookup_config(&config_tree, "ScriptsInterpreter"), &scriptinterpreter);
+	if(sandbox_can(START_PROCESSES, AFTER_SANDBOX) && sandbox_can(USE_NEW_PATHS, AFTER_SANDBOX)) {
+		free(scriptinterpreter);
+		scriptinterpreter = NULL;
+		get_config_string(lookup_config(&config_tree, "ScriptsInterpreter"), &scriptinterpreter);
+	} else {
+		logger(DEBUG_ALWAYS, LOG_NOTICE, "Not reloading ScriptsInterpreter because of sandbox.");
```

Comment:
A bit annoying if there was no `ScriptsInterpreter` set to begin with, or if it wasn't changed.